Sources/Networking/CustomerAPI.swift

```diff
@@ -106,6 +106,11 @@ final class CustomerAPI {
         let config = NetworkOperation.UserSpecificConfiguration(httpClient: self.backendConfig.httpClient,
                                                                 appUserID: transactionData.appUserID)
 
+        var receiptData = receiptData
+        if !self.backendConfig.systemInfo.dangerousSettings.usesStoreKit2JWS {
+            receiptData = receiptData.asFetchToken.asData
+        }
```

Comment:
We send the receipt base64-encoded, while the JWS token is sent as it is. I don't particularly like the placement of this piece of code, ideally we would be doing the encoding on the call site, but changing that broke a lot of receipt tests that were expecting unencoded receipts. Open to suggestions on how to handle this better.